packages/fluent/scss/core/color-system/_theme.scss

```diff
@@ -31,6 +31,11 @@ $kendo-box-shadow-depth-3: 0px 1.2px 3.6px rgba( $black, 0.1 ), 0px 6.4px 14.4px
 /// Equivalent to fluent depth 64.
 $kendo-box-shadow-depth-4: 0px 4.8px 14.4px rgba( $black, 0.18 ), 0px 25.6px 57.6px rgba( $black, 0.22 ) !default;
 
+// Validator
+$kendo-invalid-bg: null !default;
+$kendo-invalid-text: get-theme-color(error, 190) !default;
+$kendo-invalid-border: get-theme-color(error, 190) !default;
```

Comment:
I was thinking that here we should use **get-theme-color-var** function instead of **get-theme-color**. In this way, if the CSS palette var is changed by the customer, the invalid var will be changed as well. I'll update the rest of the vars in the file to use the css vars as well.  What do you think?